### DISCUSSION TITLE
time out with large pdf file

### DISCUSSION THREAD

**Discussion by kaddirov** [Stirling-Tools/Stirling-PDF#186] [Q&A] [answered]
Hi 
when i upload a large file (more then 60 MB) for compress i get a time out
![504 Gateway Time-out et 18 pages de plus - Personnel – Microsoft​ Edge](https://github.com/Frooodle/Stirling-PDF/assets/34181387/30c8d092-183c-4f09-8f70-395e8786da2b)
where can i change time out for webserver ?

**Comment by Frooodle** [comment, marked as answer]
I have increased timeout and provided overload parameter via environment variable 

https://github.com/Frooodle/Stirling-PDF/blob/main/src/main/resources/application.properties#L27

Please lemme know If this works!
